users/router.js

```diff
@@ -0,0 +1,102 @@
+const express = require('express');
+const bodyParser = require('body-parser');
+const passport = require('passport');
+
+const {User} = require('./models');
+
+const router = express.Router();
+
+const jsonParser = bodyParser.json();
+
+// Post to register a new user
+router.post('/', jsonParser, (req, res) => {
+  const requiredFields = ['username', 'password'];
+  const missingField = requiredFields.find(field => !(field in req.body));
+
+  if (missingField) {
```

Comment:
We'll have to tackle this separately, but just an FYI:  in a personal project I'm working on, I'm using [Joi](https://github.com/hapijs/joi) more or less [in this way](https://medium.com/@skwee357/validating-requests-with-a-simple-middleware-for-express-ed20c5dfd35c). Nice, clean approach to request validation.